devproxy/Dockerfile-emsdk-deps

```diff
@@ -5,6 +5,16 @@ RUN wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stab
     && tar xvf libsodium-1.0.18-stable.tar.gz && cd libsodium-stable \
     && dist-build/emscripten.sh --sumo
 
+# Build libb64
```

Comment:
does this need to be added to the `dockerfiles/Dockerfile-build-crypt-devel` and `dockerfiles/Dockerfile-build-crypt` ?

or to https://github.com/CSCfi/docker-emscripten-crypt4gh ?

Reply:
It seems from building this that `libb64` is a requirement

Reply:
It should be not required any more, since I'm dumping the public key for whitelisting as an `Uint8Array` instead of an encoded string. Removed those lines and the build requirement from `Makefile` and includes.